manim/scene/scene.py

```diff
@@ -1142,20 +1163,22 @@ def revert_to_original_skipping_status(self):
             file_writer_config["skip_animations"] = self.original_skipping_status
         return self
 
-    def add_frames(self, *frames):
+    def add_frames(self, frame, num_frames=1):
         """
         Adds a frame to the video_file_stream
 
         Parameters
         ----------
-        *frames : numpy.ndarray
-            The frames to add, as pixel arrays.
+        frame : numpy.ndarray
+            The frame to add, as a pixel array.
+        num_frames: int
+            The number of times to add frame.
         """
         dt = 1 / self.camera.frame_rate
-        self.increment_time(len(frames) * dt)
+        self.increment_time(num_frames * dt)
         if file_writer_config["skip_animations"]:
             return
-        for frame in frames:
+        for _ in range(num_frames):
```

Comment:
Are we use that this is always ever going to be used on one frame? If so, the name should also change `add_frames` -> `add_frame`.

Reply:
Yes, `num_frames` only exceeds 1 when this is called from `wait()` and the mobjects aren't moving, so I changed the function name.